sympy/solvers/tests/test_diophantine.py

```diff
@@ -1,9 +1,35 @@
-from sympy.solvers.diophantine import (diop_solve, diop_DN, diop_bf_DN, length, transformation_to_DN, find_DN, equivalent,
-    square_factor, pairwise_prime, descent,
-    ldescent, diophantine, transformation_to_normal, sum_of_four_squares, sum_of_three_squares,
-    prime_as_sum_of_two_squares, partition, power_representation)
+from sympy.solvers.diophantine import (descent,
+                                       diop_bf_DN,
+                                       diop_DN,
+                                       diop_solve,
+                                       diophantine,
+                                       divisible,
+                                       equivalent,
+                                       find_DN,
+                                       ldescent,
+                                       length,
+                                       pairwise_prime,
+                                       partition,
+                                       power_representation,
+                                       prime_as_sum_of_two_squares,
+                                       square_factor,
+                                       sum_of_four_squares,
+                                       sum_of_three_squares,
+                                       transformation_to_DN,
+                                       transformation_to_normal)
+
+from sympy import (Add,
+                   factor_list,
+                   igcd,
+                   Integer,
+                   Matrix,
+                   Mul,
+                   S,
+                   simplify,
+                   Subs,
+                   Symbol,
+                   symbols)
```

Comment:
It took me a moment to tell if various things were already imported, so I made it readable.  I'll revert it.